tests/integration/ovs_test.py

```diff
@@ -103,3 +103,17 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(
+    raises=(NmstateLibnmError, AssertionError),
```

Comment:
Does it really raise both exceptions depending on the environment? Since it is not supported at all I would expect only one exception.